dspace/config/modules/qaevents.cfg

```diff
@@ -0,0 +1,33 @@
+#---------------------------------------------------------------#
+#------- Quality Assurance Broker Events CONFIGURATIONS --------#
+#---------------------------------------------------------------#
+# Configuration properties used by data correction service      #
+#---------------------------------------------------------------#
+qaevents.solr.server = ${solr.server}/${solr.multicorePrefix}qaevent
+# A POST to these url(s) will be done to notify oaire of decision taken for each qaevents
+qaevents.openaire.acknowledge-url = https://beta.api-broker.openaire.eu/feedback/events
```

Comment:
This points to the beta version of the api, should it be https://api.openaire.eu/broker/feedback/events ?